modules/govuk/manifests/apps/service_manual_frontend.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: govuk::apps::service_manual_frontend
+#
+# Service Manual Frontend is an app to serve content pages for the service manual from the content store
+#
+# === Parameters
+#
+# [*vhost*]
+#   Virtual host used by the application.
+#   Default: 'service-manual-frontend'
+#
+# [*port*]
+#   What port should the app run on?
+#
+class govuk::apps::service_manual_frontend(
+  $vhost = 'service-manual-frontend',
+  $port = '3090',
```

Comment:
Nope, the development Procfile is how we assign ports - you're right that it feels weird but that's how we do it :). The port is used in production for the app to spin up onto and it's the place nginx will proxy requests to.